src/runtime/virtcontainers/qemu_amd64.go

```diff
@@ -51,6 +57,7 @@ var kernelParams = []Param{
 	{"reboot", "k"},
 	{"console", "hvc0"},
 	{"console", "hvc1"},
+	{"iommu", "off"},
```

Comment:
We definitely don't want this in all cases.  For one thing it will prevent usage of passed in VFIO devices with DPDK, which is something I'm working on.